sdk/src/main/java/com/deploygate/sdk/DeployGateCallback.java

```diff
@@ -6,16 +6,21 @@
  * {@link DeployGate#registerCallback(DeployGateCallback, boolean)} to listen.
  *
  * @author tnj
+ * @deprecated since 4.9.0. Use each extended callback interfaces instead.
  */
-public interface DeployGateCallback {
+@Deprecated
+public interface DeployGateCallback extends DeployGateInitializeCallback, DeployGateStatusChangeCallback, DeployGateUpdateAvailableCallback {
```

Comment:
Separate to DeployGateInitializeCallback, DeployGateStatusChangeCallback, DeployGateUpdateAvailableCallback.

Now, we marked it deprecated.
We can still use this interface, but we plan to delete it in the next major update.